doc/Methods and Results - Rough.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Methods and Results - Rough"
+author: "Farhan Majumder"
+date: "December 4, 2017"
+output: html_document
+---
+
+# Methods 
```

Comment:
A few things:
- Under dataset: our analysis didn't look at environmental variables, just at the specific habitat and feeding. All we know is that some habitats might correlate with certain envs. I also think you should add something about the sample sizes for each feeding type (and each habitat? is that too much? idk). 
- Under graph(s): This seems more fitting for a figure caption. I don't think we necessarily need to describe how we made the graph because we are providing the code and visually you can see that it's facetted.

Reply:
- I agree that sample sizes for each feeding type and habit is a good idea to disclose (maybe with a simple plot?).  You have room for more plots. Motivate your cutoff to drop groups with few samples.
- I agree that there is not need for a "Graph" section in the way it is structured right now. The text is good, but change the angle so that you describe the data and your findings *using the graphs as a visual aid*. Now you are describing the graphs, using the data as an example, which is more similar to how you would do when teaching how to make these plots. So just change the angle and flesh out the content!